helpers/githubHelper.js

```diff
@@ -143,7 +144,27 @@ class GithubHelper {
 					}
 				});
 		});
+	}
 
+	getFirstCommitForFile(fileName) {
+		const anteChronologicalOrder = (firstCommit, secondCommit) => {
+			const firstDate = new Date(firstCommit.commit.author.date);
+			const secondDate = new Date(secondCommit.commit.author.date);
+			return firstDate.getTime()-secondDate.getTime();
+		};
+		return new Promise( (resolve, reject) => {
+			this.agent
+			//not a big fan of this (even if it's server to server comms over https) ... but it allows a higher GitHub API limit
```

Comment:
Not a big fan of what?
Comments are not there to express personal preference. They are here to help others make sense of your code. It's good that you express concern, but it's more important to justify your design choices so that others know the alternatives you tried.